obs_lst_ha.pro

```diff
@@ -118,10 +120,16 @@ PRO obs_lst_ha
   ephem_lh_dat.lst_hr   =  FIX(lst_dec_hr)
   ephem_lh_dat.lst_min  =  (lst_dec_hr - FIX(lst_dec_hr))*60.0
   
-  ; Now save all this hard work
+  ; Now save ALL this hard work
   
   out_ephem_file = 'Jup_lst_ha-Nov-23-May-24.sav'
```

Comment:
We should eventually incorporate a method to prompt for or get an output name, or otherwise assign the output filename. The hardcoding method will eventually cause problems